config/runtime.exs

```diff
@@ -236,7 +236,7 @@ if config_env() == :prod do
 
   # Configure OpenID Connect
   config :openid_connect,
-    http_client_options: [ssl: FzCommon.map_ssl_opts(http_client_ssl_opts)]
+    finch_transport_opts: FzCommon.map_ssl_opts(http_client_ssl_opts)
```

Comment:
Would it make sense to set Finch as the http client [here](https://github.com/firezone/firezone/blob/master/config/config.exs#L40)? This is used [here](https://github.com/firezone/firezone/blob/master/apps/fz_http/lib/fz_http/connectivity_check_service.ex#L67) by the connectivity checks endpoint.

Sadly, the [posthog](https://github.com/nkezhaya/posthog/blob/master/lib/posthog/client.ex) library we use seems hard-coded to `hackney` :-(. Probably need to fork that at some point and customize it.

Reply:
@jamilbk yes - I plan to do that once I get to reworking connectivity checks, but it should be done in a different PR and is not critical right now

Reply:
Refs https://github.com/firezone/firezone/issues/1373